grpc/creds/creds.go

```diff
@@ -11,6 +11,24 @@ import (
 	"google.golang.org/grpc/credentials"
 )
 
+var (
```

Comment:
With entries common to both the client/server it seemed to make sense conceptually to move it to the front of the file, but I'm open to doing that in a separate PR or leaving it as-is if you feel strongly.